### PR TITLE
LazyAdam example

### PR DESCRIPTION
**Pull request by seanpmorgan**
Adding example for TFA LazyAdam. Pending feed back from new issue regarding our examples

### CODE REVIEW

File: tensorflow_addons/examples/optimizers_lazyadam.ipynb [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+      "cell_type": "markdown",
+      "source": [
+        "# TensorFlow Addons Optimizers: LazyAdam\n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/addons/blob/master/tensorflow_addons/examples/notebooks/optimizers_lazyadam.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/addons/blob/master/tensorflow_addons/examples/notebooks/optimizers_lazyadam.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (facaiy) [inline tensorflow_addons/examples/optimizers_lazyadam.ipynb]: Do we need to add `notebook` sub directory in examples directory?

Comment (seanpmorgan) [reply tensorflow_addons/examples/optimizers_lazyadam.ipynb]: I'm in favor of just removing the scripts (this PR should remove the existing one). IMO we shouldn't encourage users to import functions from examples which some might think is a good idea with scripts. Do you have a preference @facaiy?

Comment (facaiy) [reply tensorflow_addons/examples/optimizers_lazyadam.ipynb]: Sean, I think you have fixed the broken link in https://github.com/tensorflow/addons/pull/188, right? `notebooks` sub directory is not necessary, I agree :-)

Comment (seanpmorgan) [reply tensorflow_addons/examples/optimizers_lazyadam.ipynb]: Yes my fault I mis-read and quickly realized my mistake :)

**Pull request merged**